### DISCUSSION TITLE
Not a feature request, Just a "thank you" post :)

### DISCUSSION THREAD

**Discussion by andreyorst** [jocmp/capyreader#553] [Feature Requests]
Wanted to express my gratitude, because Capy is an amazing RSS reader!

I'm a huge fan of RSS and have been using a lot of readers over the years on Android. Many of them are great, but lacking some features I'd like to have. Many of them are now discontinued, and unavailable. RSS is a niche thing, so it's only natural, I guess, so my appreciation for the ones who keep it alive and kicking is humongous. Thank you for a great app!

Capy is becoming my go-to reader right now, as I finally managed to get ahold of a FreshRSS instance. Previously I was limited to local-only RSS clients, and up until you've included support for FreshRSS I wasn't sure if I should migrate to Capy from [Feeder](https://github.com/spacecowboy/Feeder). I've been using it for many years, and I have to say - it's a great app if you need a no-account RSS feed. 

I'm an extensive user of the in-app reader view, and this is the trickiest thing to get right. Feeder's reader view was almost perfect, but your implementation beats any client I've seen so far. It's amazing, and the fact that you continue to improve it is great.

I really like the simplicity of the app's interface and the fact that it doesn't try to mimic a typical social network feed. Simple and effective, pleasant to look at and interact with.

I'm looking forward to what will Capy become in the future, but IMO it's already almost perfect. So, thank you for a great app!

**Comment by jocmp** [comment]
hey thanks for the kind words!
